docs/Deploying/FAQ.md

```diff
@@ -258,8 +258,17 @@ Check out the [documentation on running database migrations](./Upgrading-Fleet.m
 
 ## What API endpoints should I expose to the public internet?
 
-If you would like to manage hosts that can travel outside your VPN or intranet we recommend only exposing the "/api/v1/osquery" endpoint to the public internet.
+If you would like to manage hosts that can travel outside your VPN or intranet we recommend only exposing the `/api/v1/osquery` endpoint to the public internet.
 
+If you are using Fleet Desktop, we recommend exposing `/api/latest/fleet/device/*/desktop`. This minimal endpoint will only provide the number of failing policies. 
```

Comment:
```suggestion
If you are using Fleet Desktop and want it to work on remote devices, the bare minimum API to expose is `/api/latest/fleet/device/*/desktop`. This minimal endpoint will only provide the number of failing policies. 

For full Fleet Desktop functionality, `/api/fleet/orbit/*` and`/api/fleet/device/ping` must also be exposed.
```

Reply:
These revisions look good to me, @ksatter. What do you think?